student.rb

```diff
@@ -1,16 +1,19 @@
-class Student
+require_relative 'People'
+
+class Student < People
+
   attr_reader :age, :phase
   attr_accessor :name
```

Comment:
These are included in all 3 original classes.  Pull up to the common superclass.